### PR TITLE
Allow logserver entries to include port numbers, with log_server_port only used if a port isn't specified.

### PR DESCRIPTION
**Pull request by greigm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -345,14 +345,14 @@ _OR_
 <pre>
 rsyslog::log_server:
   - 'log1'
-  - 'log2'
+  - 'log2:1514'
```
Comment (ghoneycutt) [inline README.md]: Add an explanation for this in the message above on line 338

**Comment by ghoneycutt** [comment]
Looks good, though we'll need some spec tests.

File: README.md (hunk 1 above)
Comment (greigm) [inline README.md]: Heh, that was a typo anyway, it should have been log3 - implying a third log server destination using a non-default port.

**Comment by greigm** [comment]
I'll try to add spec tests in the next day or so.

**Comment by greigm** [comment]
Clarified README and added spec test for log_server entries with port numbers.

**Comment by ghoneycutt** [comment]
This is failing because of updates to some gem's.. I'll take a look

**Comment by ghoneycutt** [comment]
Please rebase against master to pick up my changes that will fix travis and squash your commits.

**Comment by greigm** [comment]
Sorry, I failed at rebasing. Had a bad day. ;) All done.

**Comment by ghoneycutt** [comment]
Thanks!

**Pull request merged**